common/utilities/config.py

```diff
@@ -0,0 +1,24 @@
+"""
+Config
+
+This module holds the config used by an application. To use this module, first
+call `setup_config` to populate `config`. Then, just get any required config
+from `config`.
```

Comment:
The design decision on configuration says:

> there is no need to write code to handle the cases where the config item hasn't been provided, simply erroring with a clear message detailing which required configuration item is unavailable is sufficient

Should we implement a `get_config` function that handles logging a standard message if the requested configuration value is not available, or was that deliberately ruled out?